kalite/topic_tools/__init__.py

```diff
@@ -300,7 +300,8 @@ def get_content_cache(force=False, annotate=False, language=None):
             CONTENT[language] = content
             return CONTENT[language]
         else:
-            CONTENT[language] = softload_json(settings.CONTENT_FILEPATH, logger=logging.debug, raises=False)
+            CONTENT[language] = (softload_sqlite_cache(settings.CONTENT_CACHE_FILEPATH) or
```

Comment:
OK, let's merge this, pending further testing during the debug period.

Might be worth having a setting that forces the management command to execute when the caches are initially generated, and have it actually make a commit during annotation to prevent annotation being required.